source/features/next-scheduled-github-action.tsx

```diff
@@ -54,6 +54,10 @@ async function init(): Promise<false | void> {
 	}
 
 	for (const workflowListItem of select.all('[href*="?query"]', await elementReady('.hx_actions-sidebar'))) {
+		if (select('svg', workflowListItem)?.classList.contains('octicon-stop')) {
```

Comment:
I am not in front of a computer not but would a css idea like this one work here?

https://github.com/sindresorhus/refined-github/blob/6651e05638b1b6609048dcc8799a043448834eb0/source/features/bypass-checks.tsx#L20-L24

Reply:
We could loop directly over the SVGs in the workflows list (excluding the ones with `octicon-stop` class), use `nextSibling()` instead of `textContent` and `insertAdjacentElement('afterend', x)` instead of `append()`.

Thoughts @fregante ?

Reply:
Can you look to see if there is any css solution?

Reply:
By "css solution", do you mean hide the inserted info using a selector like `.hx_actions-sidebar svg:not(.octicon-stop) ~ em`?

If we're going to rely on the SVGs anyway, wouldn't it be better to ignore the disabled workflows directly in the loop instead of computing the info and then hiding it?

Reply:
He means in a single CSS selector. Use `select.exists('.octicon-stop', item)`

Reply:
You’re missing part of the code

Reply:
Sorry about that, fixed it.